pages/understanding-json-schema/reference/enum.md

```diff
@@ -18,16 +18,21 @@ The following is an example for validating street light colors:
 ```json
 // props { "isSchema": true }
 {
-  "enum": ["red", "amber", "green"]
```

Comment:
I suggest we should keep this example and add yours to the document instead of replacing examples.

Reply:
This is just my suggestion I left the final decision to the author